backend/src/test/kotlin/org/loculus/backend/controller/submission/GetReleasedDataEndpointTest.kt

```diff
@@ -343,6 +351,51 @@ class GetReleasedDataEndpointTest(
         assertThat(data[0].metadata, `is`(not(emptyMap())))
     }
 
+    /**
+     * This test ist relevant for EarliestReleaseDateFinder -- It relies on this particular ordering to be returned.
```

Comment:
```suggestion
     * This test ist relevant for EarliestReleaseDateFinder which relies on this particular ordering to be returned.
```

Otherwise, one could read it as that the test relies on the particular ordering.